zebrad/build.rs

```diff
@@ -55,4 +55,10 @@ fn main() {
             vergen(config).expect("non-git vergen should succeed");
         }
     }
+
+    tonic_build::configure()
```

Comment:
PR failure: This build command seems to expect `rustfmt`, but it's not installed in our docker image.

We can add it, but the `rustfmt` requirement seems a bit odd.

> error: failed to run custom build command for `zebrad v1.0.0-beta.7 (/app/zebrad)`
> error: 'rustfmt' is not installed for the toolchain '1.60.0-x86_64-unknown-linux-gnu'

https://github.com/ZcashFoundation/zebra/runs/5890360984?check_suite_focus=true#step:9:1614

Reply:
Yeah, that's not great :/

I could have disabled the `rustfmt` feature, but thought it was best to just upgrade to the newest version which [doesn't use `rustfmt` anymore](https://github.com/hyperium/tonic/pull/904).